### PR TITLE
Center align spinner

### PR DESCRIPTION
**Pull request by neelxie**
#### What does this PR do?
Fix spinner alignment and no messages message

#### Ticket
https://trello.com/c/HsyelwYh

#### Screenshot
**Before**
Screenshot from 2021-05-24 19-31-24

**After**
Screenshot from 2021-05-24 19-30-53

### CODE REVIEW

File: src/components/DatabaseList/DatabaseList.module.css [hunk 1]
```diff
@@ -94,4 +94,5 @@
   text-align: center;
   justify-content: center;
   align-items: center;
+  padding-top: 15%;
```
Comment (kawerewagaba) [inline src/components/DatabaseList/DatabaseList.module.css:97]: do u mind giving more context on this

**Review by kawerewagaba — CHANGES_REQUESTED** [review]
i'd like to understand your implementation

**Review by Mubangizi — COMMENTED** [review]
Hey @neelxie, I think the spinner should be moved a little bit down from the header table. It looks like its too into the table

File: src/components/DatabaseList/DatabaseList.module.css (hunk 1 above)
Comment (neelxie) [reply src/components/DatabaseList/DatabaseList.module.css]: @kawerewagaba That was wrong I have removed it.

**Comment by neelxie** [comment]
> Hey @neelxie, I think the spinner should be moved a little bit down from the header table. It looks like its too into the table

Screenshot from 2021-06-24 15-00-43
@Mubangizi is this sufficient?

**Review by kawerewagaba — APPROVED** [review]
lgtm

**Pull request merged**